### PR TITLE
Support non-1000 user ids on linux with local mode

### PR DESCRIPTION
**Pull request by ryanmelt**

### CODE REVIEW

File: compose.yaml [hunk 1]
```diff
@@ -117,14 +112,13 @@ services:
     environment:
       - "RAILS_ENV=production"
       - "GEM_HOME=/gems"
-    group_add:
-      - ${OPENC3_LOCAL_MODE_GROUP_ID}
```
Comment (jmthomas) [inline compose.yaml:121]: By default, the docker container process runs with the supplementary groups looked up for the specified user. So is that why we can remove this?

File: openc3-cosmos-cmd-tlm-api/Dockerfile [hunk 2]
```diff
@@ -16,11 +16,10 @@ RUN bundle config set --local without 'development' \
     /var/tmp/*
 
 RUN ["chown", "-R", "openc3:openc3", "/src/"]
```
Comment (jmthomas) [inline openc3-cosmos-cmd-tlm-api/Dockerfile:18]: Is this needed if we're just going to chmod 777 in two lines?

File: openc3.code-workspace [hunk 3]
```diff
@@ -2,54 +2,6 @@
...
-      "path": "../openc3-widget-template"
-    },
-    {
-      "path": "../openc3-microservice-template"
-    },
-    {
-      "path": "../cosmos-project"
-    },
-    {
-      "path": "../cosmos-enterprise-project"
```
Comment (jmthomas) [inline openc3.code-workspace:52]: Why remove?

**Review by jmthomas — APPROVED** [review]

**Comment by jmthomas** [comment]
Not sure why trivy is timing out. Nothing has changed and the other images finish in seconds.

File: compose.yaml (hunk 1 above)
Comment (ryanmelt) [reply compose.yaml:121]: The container now runs as the user and the users default group.  So adding additional groups isn't necessary.

File: openc3-cosmos-cmd-tlm-api/Dockerfile (hunk 2 above)
Comment (ryanmelt) [reply openc3-cosmos-cmd-tlm-api/Dockerfile:18]: It still sets the expected owner.

File: openc3.code-workspace (hunk 3 above)
Comment (ryanmelt) [reply openc3.code-workspace:52]: We shouldn't be checking all this stuff in to the repo.  I added developer.code-workspace to .gitignore.  You and I should use a file with that name and not clutter this for everyone else.

**Pull request merged**